zipkin-lens/src/components/common/ServiceBadge.test.js

```diff
@@ -12,34 +12,33 @@
  * the License.
  */
 import React from 'react';
-import { mount } from 'enzyme';
 
 import ServiceBadge from './ServiceBadge';
+import { render, screen } from '@testing-library/react';
```

Comment:
Use `@testing-library/react` instead of enzyme since all other test code already uses it.